src/cmakeFindModules/FindOpenMVG.cmake

```diff
@@ -29,32 +29,46 @@ ELSE()
   MESSAGE(STATUS "NOT FOUND")
 ENDIF (OPENMVG_INCLUDE_DIR)
 
-SET(OPENMVG_LIBRARIES_NAMES  
-  openMVG_numeric
-  openMVG_system
+SET(OPENMVG_LIBRARIES_NAMES
```

Comment:
To have the same list in `src/CMakeLists.txt``and `FindOpenMVG.cmake`.
We have just commented `openMVG_linearProgramming` because it creates problems and we have not checked why.